torch/nested/_internal/nested_tensor.py

```diff
@@ -20,12 +20,22 @@ def get_tensor_id(tensor, factor):
 class NestedTensor(torch.Tensor):
     _values: torch.Tensor  # type: ignore[assignment]
     _offsets: torch.Tensor
-    _size: Tuple[int, int, int]
+    _size: Tuple[int, torch.SymInt, int]
+    ragged_size: torch.SymInt
```

Comment:
doc here would be good. Why isn't this underscored?

Reply:
Underscored, doc coming soon.